source/form.html

```diff
@@ -0,0 +1,174 @@
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button">
+        <span class="visually-hidden">Открыть меню.</span>
+      </button>
+      <ul class="site-list">
+        <li class="site-list__item">
+          <a href="index.html">
+            Главная
+          </a>
+        </li>
+        <li class="site-list__item">
+          <a href="catalog.html">
+            Каталог продукции
+          </a>
+        </li>
+        <li class="site-list__item site-list__item--active">
+          <a>
+            Подбор программы
+          </a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="form-headline">
+    <h1 class="form-headline__title">Подбор программы</h1>
+    <p class="form-headline__offer">Заполните анкету, и мы подберем<br>программу питания для
+      вашего кота</p>
+    <form class="form" action="https://echo.htmlacademy.ru/" method="post">
+      <fieldset class="form__field-name-weight-age">
+        <label class="form__name" for="name">
+          Имя:*
+        </label>
+        <input class="form__input" type="text" placeholder="БАРСИК" name="cat-name" required id="name">
+        <label class="form__weight" for="weight">
+          Вес(кг):*
+        </label>
+        <input class="form__input" type="number" placeholder="7" name="cat-weight" required id="weight">
+        <label class="form__age" for="age">
+          Возраст(лет):
+        </label>
+        <input class="form__input" type="number" name="cat-age" placeholder="7" id="age">
+      </fieldset>
+      <fieldset class="form__field-radio">
+        <ul class="radio-button">
+          <li class="radio-button__weight-loss">
+            <label class="control">
+              <input class="control__input" type="radio" name="target-group" value="weight-loss" checked>
+              <span class="control__label">Похудение</span>
+            </label>
+          </li>
+          <li class="radio-button__weight-gain">
+            <label class="control">
+              <input class="control__input" type="radio" name="target-group" value="weight-gain">
+              <span class="form__control-label">Набор массы</span>
+            </label>
+          </li>
+          <li class="radio-button__need-help">
+            <label class="control">
+              <input class="control__input" type="radio" name="target-group" value="need-help">
+              <span class="control__label">Не знаю (нужен ваш совет)</span>
+            </label>
+          </li>
+        </ul>
+      </fieldset>
+      <fieldset class="form__contact">
+        <legend class="form__contact-title">Контактные данные<br>(владельца кота)</legend>
+        <label class="form__contact-email" for="email">
+          E-MAIL:*
+        </label>
+        <input class="form__contact-input" type="email" placeholder="kuklachev@gmail.com" name="email" required
+          id="email">
+        <label class="form__contact-nomber" for="nomber">
```

Comment:
```suggestion
        <label class="form__contact-nomber" for="number">
```

nomber - нет такого слова

Reply:
наступил второй раз на грабли